target/linux/ath79/nand/base-files/etc/hotplug.d/firmware/10-ath9k-eeprom

```diff
@@ -12,6 +12,7 @@ case "$FIRMWARE" in
 	8dev,rambutan)
 		caldata_extract "caldata" 0x1000 0x800
 		;;
+	netgear,r6100|\
```

Comment:
No (valid) MAC address in 0x1002?

Reply:
There is, but I guess we still need to extract the caldata contents.

Reply:
Yes, but then you don't need to overwrite it with nvmem-cells in DTS (at &wmac).